treeherder/webapp/api/jobs.py

```diff
@@ -375,6 +375,7 @@ def list(self, request, project):
 
         return Response(response_body)
 
+    # TODO remove
```

Comment:
Do you want to keep this comment?

Reply:
yes. I'll remove it in the final pr to remove the step table.